src/App.js

```diff
@@ -1,19 +1,23 @@
 import React from 'react';
 import TodoList from './TodoList';
 import AddTodoForm from './AddtodoForm';
+import "./style.css"
 
 function App() {
 
-  let [newTodo, setNewTodo] = React.useState('');
+  let [todoList, setTodoList] = React.useState([]);
+
+  function addTodo (newTodo){
+    setTodoList(existingTodoList => [...existingTodoList, newTodo]);
```

Comment:
Great job here but you do not need to add include a function here. You can do it like `[...todoList, newTodo]`